### ISSUE TITLE
Unable to build Dockerfile.

### ISSUE THREAD

**Issue by jason-fox** [Engineering-Research-and-Development/o2k-connector#1]
Running Docker buildx with the following command:

```console
docker buildx build --platform linux/amd64,linux/arm64 --push -t quay.io/fiware/o2k-connector:1.0.0 .
```

An Error occurs when installing requirements:

```
 => ERROR [linux/arm64  5/10] RUN pip install -r requirements.txt                                                                                   18.5s
 => CANCELED [linux/amd64  5/10] RUN pip install -r requirements.txt  


...


18.03   Running setup.py install for confluent-kafka: started
18.24   Running setup.py install for confluent-kafka: finished with status 'error'
18.24   error: subprocess-exited-with-error
18.24   
18.24   × Running setup.py install for confluent-kafka did not run successfully.



18.24       In file included from /tmp/pip-install-5vxiv_7z/confluent-kafka_53c2fe66a83440499ba67edecedc092d/src/confluent_kafka/src/Admin.c:17:
18.24       /tmp/pip-install-5vxiv_7z/confluent-kafka_53c2fe66a83440499ba67edecedc092d/src/confluent_kafka/src/confluent_kafka.h:23:10: fatal error: librdkafka/rdkafka.h: No such file or directory
18.24          23 | #include <librdkafka/rdkafka.h>
18.24             |          ^~~~~~~~~~~~~~~~~~~~~~
18.24       compilation terminated.
18.24       error: command '/usr/bin/gcc' failed with exit code 1
18.24       [end of output]
18.24   
18.24   
18.24   note: This error originates from a subprocess, and is likely not a problem with pip.
18.24 error: legacy-install-failure
18.24 
18.24 × Encountered error while trying to install package.
18.24 ╰─> confluent-kafka
18.24 
18.24 note: This is an issue with the package mentioned above, not pip.
18.24 hint: See above for output from the failure.
18.25 WARNING: You are using pip version 22.0.4; however, version 23.2.1 is available.
18.25 You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
------
Dockerfile:8
--------------------
   6 |     
   7 |     COPY requirements.txt requirements.txt 
   8 | >>> RUN pip install -r requirements.txt
   9 |     
  10 |     RUN mkdir /o2k-connector
--------------------
ERROR: failed to solve: process "/bin/sh -c pip install -r requirements.txt" did not complete successfully: exit code: 1

 ```

**Comment by manfredipist** [comment]
Fixed on v 1.0.1

**Comment by jason-fox** [comment]
@manfredipist - why has the naming convention for the images changed to accommodate the architectures? Shouldn't that be part of the `arch` of the overall image? You can place multiple supported architectures under one tag.

I'd like to know how you are planning to do this going forward so that we (FF) can continue to clone your images as the **source-of-truth**

**Comment by manfredipist** [comment]
I couldn't build for multi-arch, it is now solved and tag 1.0.1 has been published
